src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -431,7 +431,7 @@ public void buttonCvrFilePathClicked() {
       case CDF -> {
         FileChooser fc = new FileChooser();
         fc.setInitialDirectory(new File(FileUtils.getUserDirectory()));
-        fc.getExtensionFilters().add(new ExtensionFilter("JSON files", "*.json"));
+        fc.getExtensionFilters().add(new ExtensionFilter("JSON and XML files", "*.json", "*.xml"));
```

Comment:
Oops, one more thing here... you need to un-disable the "Contest ID" field in the GUI when CDF is selected. To do this, look for this line:
```
case CLEAR_BALLOT, DOMINION, HART
```
And add `CDF` in there, and delete the clause above it that currently handles CDF separately.